easybuild/easyconfigs/l/libfabric/libfabric-1.15.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'ConfigureMake'
+
+name = 'libfabric'
+version = '1.15.1'
+
+homepage = 'https://ofiwg.github.io/libfabric/'
+description = """
+Libfabric is a core component of OFI. It is the library that defines and exports
+the user-space API of OFI, and is typically the only software that applications
+deal with directly. It works in conjunction with provider libraries, which are
+often integrated directly into libfabric.
+"""
+
+# The psm3 provider (enabled by default) requires an AVX capable system to run
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ofiwg/libfabric/releases/download/v%(version)s']
+sources = [SOURCE_TAR_BZ2]
+patches = ['%(name)s-1.12.1_disable-psm3-avx-config.patch']
+checksums = [
+    'cafa3005a9dc86064de179b0af4798ad30b46b2f862fe0268db03d13943e10cd',  # libfabric-1.15.1.tar.bz2
+    # libfabric-1.12.1_disable-psm3-avx-config.patch
+    '129be05a824b41eeb7bddac3c23aa421a13f7dde622948df942a265ccc96125a',
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('pkgconf', '1.8.0'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('numactl', '2.0.14'),
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+# Regenerate build files to pick up psm3-axv-config patch
+preconfigopts = "autoreconf -if &&"
```

Comment:
I'd recommend
`preconfigopts = "./autogen.sh &&"`
here instead. While it will do the same thing in the end, if such a script exists we should run it, as it's the "upstream blessed" way of regenerating autotools files.

Reply:
good point, updated